packages/framework/src/Resources/config/parameters_common.yml

```diff
@@ -8,5 +8,8 @@ parameters:
     # Administration locale. Must be registered in domains.yml
     shopsys.admin_locale: en
 
+    # All time values in administration are displayed in this timezone (default is PHP timezone)
+    shopsys.display_timezone: ~
```

Comment:
I would prefer if it was set here to prague

Reply:
I cannot set it here to Prague, because that could change the times for someone who upgrade to the newest version (My server is set to 'America/Phoenix' and I count with that in application). Setting it here means their time shifts on the frontend.

I set it in `ShopBundle` `parameters_common.yml` file only on purpose so new installations are in Europe/Prague (acceptance criteria), but upgrade does not change a thing because default is server timezone.

Reply:
Ok good point.

Reply:
You forgot to update parameters_common in project-base

Reply:
it's in different commit – 9cadeb4442c9a351daa297cd99a192ed19aa0e62